src/components/swap/SwapModalFooter.tsx

```diff
@@ -209,3 +217,30 @@ export default function SwapModalFooter({
     </>
   )
 }
+
+function TokenTaxLineItem({ trade, type }: { trade: ClassicTrade; type: 'input' | 'output' }) {
+  const [currency, percentage] =
+    type === 'input' ? [trade.inputAmount.currency, trade.inputTax] : [trade.outputAmount.currency, trade.outputTax]
+
+  const color = useWarningColor(percentage)
+
+  if (percentage.equalTo(ZERO_PERCENT)) return null
+
+  return (
+    <ThemedText.BodySmall>
+      <Row align="flex-start" justify="space-between" gap="sm">
+        <MouseoverTooltip
+          text={
+            <Trans>
+              Some tokens take a fee when they are bought or sold, which is set by the token issuer. Uniswap does not
+              receive any share of these fees.
```

Comment:
See language nits in other comment - both on this sentence, and on the presentation of the fee.